recipes/pandarallel/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "pandarallel" %}
+{% set version = "1.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pandarallel-{{ version }}.tar.gz
+  sha256: 3fa7d9baf79ad49e71db923a924a127f2b06a2b45d7229b47655fbcb620929b2
+
+build:
+  number: 0
+  skip: true  # [win]
```

This looks like it can be `noarch: python`:

```suggestion
  noarch: python
```

---

The lib does not work on Windows system. It means people will be able to install it but it won't work. Is that still ok?

---

I think that’s fine I’m this case